app/screens/channel_info/channel_info.js

```diff
@@ -351,7 +351,17 @@ export default class ChannelInfo extends PureComponent {
         } = this.props;
 
         if (channelIsArchived) {
-            return null;
+            return (this.renderViewOrManageMembersRow() &&
+                <View>
+                    <ChannelInfoRow
+                        action={this.goToChannelMembers}
+                        defaultMessage={'View Members'}
+                        detail={currentChannelMemberCount}
+                        icon='users'
+                        textId={t('channel_header.viewMembers')}
+                        theme={theme}
+                    />
+                </View>);
```

Comment:
On screenshot, there's a missing separator between View Members and Leave Channel.

Reply:
@saturninoabril Added separator line and updated screenshot. Thanks.